ingest/clusters.py

```diff
@@ -46,30 +48,30 @@ def transform(self, row):
 
             # first index is cell name don't need to check annot type
             if idx != 0:
-                if self.metadata_types[idx].casefold() == 'numeric':
+                if self.metadata_types[idx].casefold() == "numeric":
                     column = round(float(column), 3)
-                elif self.metadata_types[idx].casefold() == 'group':
+                elif self.metadata_types[idx].casefold() == "group":
                     if column not in self.unique_values[annotation]:
                         self.unique_values[annotation].append(column)
             # perform a shallow copy
-            annotation_value = copy.copy(self.cluster_subdocs[annotation]['value'])
+            annotation_value = copy.copy(self.cluster_subdocs[annotation]["value"])
             annotation_value.append(column)
-            self.cluster_subdocs[annotation]['value'] = annotation_value
+            self.cluster_subdocs[annotation]["value"] = annotation_value
 
     def return_cluster_subdocs(self, headers):
         """Creates cluster_subdocs"""
         cluster_subdocs = {}
         for annot_name in self.header:
             value = annot_name.lower()
-            if value == 'name':
-                cluster_subdocs[value] = self.create_cluster_subdoc('text', 'cells')
-            elif value in ('x', 'y', 'z'):
+            if value == "name":
+                cluster_subdocs[value] = self.create_cluster_subdoc("text", "cells")
+            elif value in ("x", "y", "z"):
                 cluster_subdocs[value] = self.create_cluster_subdoc(
-                    value, 'coordinates'
+                    value, "coordinates"
                 )
             else:
                 cluster_subdocs[value] = self.create_cluster_subdoc(
```

Comment:
this should be:

`cluster_subdocs[annot_name] = self.create_cluster_subdoc(...)`

Otherwise we still get the same `KeyError` as before.  I tested this locally and it works.

Reply:
addressed in 987cc80